ol_stock_constrained_availability/models/mrp_production_batch.py

```diff
@@ -0,0 +1,86 @@
+# Import Odoo Libs
+from odoo import models, fields
+from odoo.tools import float_compare
+from odoo.tools.misc import format_date
```

Comment:
FYI: I added this import during my code formatting changes as I noticed it complaining that this was missing.

Reply:
Yes, you are right; it was removed while pushing code. I don't know how that happened. Which formatting are you using so I can use that one?